public/help/features.md

```diff
@@ -1,6 +1,6 @@
 # Features
 
-This is a non-exhaustive, non-priortised list of features that JS Bin has.
+This is a non-exhaustive, non-prioritized list of JS Bin features.
```

Comment:
There's no `z` in prioritised. Definitely a typo in there, but yeah, we're not a US company.